package.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from '@sanity/pkg-utils';
+
+export default defineConfig({
+  dist: 'lib',
+  minify: true,
+  legacyExports: true,
+  // Remove this block to enable strict export validation
+  extract: {
+    rules: {
+      'ae-forgotten-export': 'off',
+      'ae-incompatible-release-tags': 'off',
+      'ae-internal-missing-underscore': 'off',
+      'ae-missing-release-tag': 'off',
```

Comment:
I copied this from [sanity-plugin-media@v3](https://github.com/sanity-io/sanity-plugin-media/blob/v3/package.config.ts). Not sure if we want this rules enabled

Reply:
No rush to enable these atm, but they will come into play once we get around to shipping automated docs pages based on ts-docs in plugin-kit.